web_src/js/features/projects.js

```diff
@@ -87,13 +149,113 @@ export default async function initProject() {
       data: JSON.stringify({title: boardTitle.val()}),
       headers: {
         'X-Csrf-Token': csrf,
-        'X-Remote': true,
+        'X-Remote': true
       },
       contentType: 'application/json',
-      method: 'POST',
+      method: 'POST'
     }).done(() => {
       boardTitle.closest('form').removeClass('dirty');
       setTimeout(window.location.reload(true), 2000);
     });
   });
+
+  $('#new-project-issue-item').on('click', (_e) => {
+    $('#current-card-details').removeClass('hide');
+    $('#current-card-details').css('visibility', 'visible');
+
+    $('#current-card-details').show();
+    const searchHtml = `
+      <div class='ui search'>
+        <input class='prompt' type='text' id='issue-search' placeholder='Filter issues'/>
+      </div>
+    `;
+    const html = `
+      <div id='issue-results' class='ui cards draggable-cards'></div>
+    `;
+    $('#current-card-details').html(html);
+    $('#current-card-details-input').html(searchHtml);
+    sortCards('#issue-results');
+  });
+
+  function sortCards(selector) {
+    $(selector).each((_i, eli) => {
+      new Sortable(eli, {
+        group: 'shared',
+        filter: '.ignore-elements',
+        animation: 150,
+        onEnd(/** Event*/ e) {
+          const cardsFrom = [];
+          const cardsTo = [];
+          $(e.from).each((_i, v) => {
+            const column = $($(v)[0]).data();
+            $(v)
+              .children()
+              .each((j, y) => {
+                const card = $(y).data();
+                if (card && card.id && e.oldDraggableIndex !== e.newDraggableIndex) {
+                  cardsFrom.push({id: card.id, priority: j, ProjectBoardID: column.columnId
+                  });
+                }
+              });
+          });
+
+          $(e.to).each((_i, v) => {
+            const column = $($(v)[0]).data();
+            $(v)
+              .children()
+              .each((j, y) => {
+                const card = y.dataset;
+                if (card && card.id) {
+                  cardsTo.push({
+                    id: parseInt(card.id),
+                    priority: parseInt(j),
+                    ProjectBoardID: parseInt(column.columnId),
+                    issueid: parseInt(card.issueid),
+                    projectID: parseInt(projectID)
+                  });
+                } else if (card && card.issueid && card.id === 0) {
+                  cardsTo.push({
+                    issueid: parseInt(card.issueid),
+                    priority: parseInt(j),
+                    ProjectBoardID: parseInt(column.columnId),
+                    projectID: parseInt(projectID)
+                  });
+                }
+              });
+          });
+
+          // Can't use await because onEnd is not async
```

Comment:
Does `onEnd` expect a synchronous return value? I think not so there's no reason to not make it async 😉